content/sensu-go/6.8/catalog/catalog-reference.md

```diff
@@ -0,0 +1,1229 @@
+refFilter: .labels.provider == "alerts"
+{{< /code >}}
+
+title        | 
+-------------|------ 
+description  | Label to display above the user input field.
+required     | true
+type         | String
+example      | {{< code yml >}}
+title: NGINX stub status URL
+{{< /code >}}
+
+type           | 
+---------------|------ 
+description    | Type of input requested.
+required       | true
+type           | String
+allowed values | `boolean`, `integer`, `string`
+example        | {{< code yml >}}
+type: string
+{{< /code >}}
+
+#### Patches attributes
+
+op           | 
+-------------|------ 
+description  | Patch operation to perform.
+required     | false
+type         | String
+allowed values | `add`, `replace`
+example      | {{< code yml >}}
+op: replace
+{{< /code >}}
+
+path         | 
+-------------|------ 
+description  | Path for the attribute to patch within the specified Sensu resource. In [JSON Pointer][14] format, which supports array indexes such as `/spec/subscriptions/0`. Use `-` to insert values at the end of an array (for example, `/spec/subscriptions/-`).
+required     | false
+type         | String
+example      | {{< code yml >}}
+path: /spec/subscriptions
+{{< /code >}}
+
+value        | 
+-------------|------ 
+description  | Built-in or user-entered value to apply in the patch. The built-in value is `unique_id`, which randomly generates an 8-digit hexadecimal string value (e.g. 168c41a1). User-entered variables are represented by variable name references in double square brackets.
+required     | false
+type         | 
+example      | {{< code yml >}}
+value: [[ subscriptions ]]
+{{< /code >}}
+
+##### Resource attributes
+
+api_version  | 
+-------------|------ 
+description  | Sensu API group and version for the resource to patch.
+required     | true
+type         | String
+example      | {{< code yml >}}
+api_version: core/v2
+{{< /code >}}
+
+name         | 
+-------------|------ 
+description  | Name of the resource to patch.
+required     | true
+type         | String
+example      | {{< code yml >}}
+name: nginx-metrics
+{{< /code >}}
+
+type         | 
+-------------|------ 
+description  | Type of the resource to patch.
+required     | true
+type         | String
+example      | {{< code yml >}}
+type: CheckConfig
+{{< /code >}}
+
+## Resource limits
+
+There is no limit on the number of resources you can bundle into a single integration.
+Each integration can include as many checks, event filters, handlers, and pipelines as you need to achieve your observability goals.
+For example, you can develop a single host monitoring integration that includes all of the checks you want to run on every server.
+
+## Check guidelines
+
+For integrations that create checks, list the resource definitions in your `sensu-resources.yaml` file in in the following order:
+
+1. CheckConfig
+2. HookConfig
+3. Secret
+4. Asset
+
+The prompts for check resources should include at least one subscription.
```

Comment:
Caleb's catalog notes at https://github.com/sensu/catalog#sensu-integration-guidelines-1 say this is a requirement, but there are several integrations in the official catalog that do not include or require a subscription (the checks would require one to run obviously, but the integration workflow itself does not require providing a subscription to install the integration).

Reply:
Whether that subscription is hardcoded into the integration or requested via a prompt, a check resource needs at least one subscription to be created. I think using "should" in the context of prompts is ok.